_rules/aria-required-context-role-ff89c9.md

```diff
@@ -84,7 +86,7 @@ These elements with an [explicit role][] of `listitem` are [owned by][] an eleme
 
 #### Passed Example 3
 
-These elements with an [explicit role][] of `listitem` are [owned by][] an element with their [required context role][] even though they are not direct DOM children of it. ARIA [ownership][owned by] "skips" through presentational nodes because they are not [included in the accessibility tree][].
+These elements with an [explicit role][] of `listitem` are children in the [accessibility tree][] of an element with their [required context role][] even though they are not direct DOM children of it. The presentational nodes are not [included in the accessibility tree][], hence the two innermost `div` are children of the outermost in the [accessibility tree][] (but not in the DOM tree).
```

Comment:
```suggestion
These elements with an [explicit role][] of `listitem` are children in the [accessibility tree][] of an element with their [required context role][] of `list`, even though they are not its direct children in DOM. The presentational node is not [included in the accessibility tree][].
```
It kind of read to me like `div`s are included in the accessibility tree and not the `listitem``s. I think it's well explained without the last sentence.